tests/ui/parser/raw/raw-idents.rs

```diff
@@ -0,0 +1,157 @@
+//@ check-pass
+// To make everything a keyword:
+//@ edition:2024
```

Comment:
Question: should we revision this test so it checks editions (2015, 2018, 2024)?

Reply:
Sure, why not :3